driver-sync/src/test/functional/com/mongodb/client/ConnectivityTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client;
+
+import org.bson.Document;
+import org.junit.Test;
+
+public class ConnectivityTest {
+
+    @Test
+    public void testConnectivity() {
+        MongoClient client = MongoClients.create(Fixture.getMongoClientSettings());
+
+        try {
+            // test basic connectivity
+            client.getDatabase("admin").runCommand(new Document("ismaster", 1));
+
+            // test authenticated connectivity
+            client.getDatabase("test").getCollection("test").count();
```

Comment:
Should we assert anything here? Perhaps add a catch block: 

```
} catch (Exception e) {
    fail(format("Connectivity failed: %s", e.getMessage()));
}
```

It makes the intent of the test clearer if nothing else

Reply:
The nice thing about the way it is now is that as part of the test failure (due to an exception leaking out) Junit prints the entire stack trace, which makes debugging easier than if the test catches and calls fail with just the message.

I'll add a comment explaining the intent better.  Let me know if that suits you.